### ISSUE TITLE
compilation failing

### ISSUE THREAD

**Issue by oivulf** [rdkit/rdkit#2432]
**Description:**
Release: 2019_03_01
OS: Linux Debian Buster
Compiler: g++ 8.3.0

Compilation error:
/home/fc/build/rdkit-Release_2019_03_1/Code/GraphMol/FileParsers/MaeMolSupplier.cpp:48:59: error: no matching function for call to ‘schrodinger::mae::Reader::Reader(std::shared_ptr<std::basic
_istream<char> >&)’       
   d_reader.reset(new schrodinger::mae::Reader(dp_sInStream));

I guess the culprit is this note:

/usr/include/maeparser/Reader.hpp:27:5: note: candidate: ‘schrodinger::mae::Reader::Reader(std::istream&, size_t)’                                                                            
     Reader(std::istream& stream,
     ^~~~~~
/usr/include/maeparser/Reader.hpp:27:5: note:   no known conversion for argument 1 from ‘std::shared_ptr<std::basic_istream<char> >’ to ‘std::istream&’ {aka ‘std::basic_istream<char>&’}

**Comment by oivulf** [comment]
Failure also with clang, at the same place.

**Comment by bp-kelley** [comment]
For some reason you are picking up an maeparser in /usr/include 

Rdkit should be using it’s own maeparser, I expect this is an unusual situation caused by /usr/include being first in the include search path.

**Comment by oivulf** [comment]
Ah, thank you, I had not noticed that. I can cope with it now.

**Comment by adalke** [comment]
I had the same problem. This discussion helped me narrow down where to look. Turns out I was building RDKit from a new pull from GitHub, in a directory where I pulled/built RDKit last fall. The build used the old unpacked directory in External/CoordGen/ .

I did a:
```
cd External/CoordGen/
rm -rf coordgenlibs-1.3.tar.gz coordgenlibs/ maeparser/
```

and the cmake step again (to download&install the updated code), then the code built.
